packages/gpu_voxels/CMakeLists.txt

```diff
@@ -199,7 +199,7 @@ ICMAKER_CONFIGURE_PACKAGE()
 
 ###############################################################################
 # Build examples
-ADD_SUBDIRECTORY(src/examples)
+# ADD_SUBDIRECTORY(src/examples)
```

Comment:
Did you disable this to save compilation time or because of errors?
I would prefer introducing a CMake build option for this.

Reply:
As mentioned before, no issues was found. I agree with that suggestion.